projects/igniteui-angular/src/lib/grids/common/crud.service.ts

```diff
@@ -568,7 +559,6 @@ export class IgxGridCRUDService extends IgxRowAddCrudState {
 
             } else {
                 this.createCell(cell);
-                this.grid.onFormGroupCreate.emit(this.cell.row.rowFormGroup);
```

Comment:
We might actually need to keep this place in the process rather than general init, just to make the validity state adequate, but will come back to you on this.

Reply:
Actually you might need to hook into the end edits just to know when an edit passes to keep the validity for it